src/shipit_risk_assessment/requirements.txt

```diff
@@ -1,3 +1,4 @@
 -e ./../../lib/cli_common[pulse,taskcluster,mercurial] #egg=mozilla-cli-common
 
 # mozilla-mocoda when it will be on PyPi
+-e https://github.com/mozilla/mocoda/archive/fb04724fd484bfcc949216decc15e2c2467b2f30.tar.gz #egg=mozilla-mocoda
```

Comment:
I think you need to run ``make update-app APP=shipit-risk-assessment`` to also update ``requirements.nix``.